### DISCUSSION TITLE
How to use PnP Core with System-assigned managed identity?

### DISCUSSION THREAD

**Discussion by jaganpt** [pnp/pnpcore#456] [Q&A] [answered]
I have a azure function app with system-assigned managed identity enabled and added the api permissions through power shell as per the documentation below.
https://docs.microsoft.com/en-us/azure/app-service/overview-managed-identity?tabs=dotnet

Now I would like to use PnP Core SDK (https://github.com/pnp/pnpcore)  and authenticate using System-assigned managed identity. How to use PnP Core with System-assigned managed identity?

**Comment by jansenbe** [comment, marked as answer]
Tracked via https://github.com/pnp/pnpcore/issues/457
